data-model/src/main/java/io/micronaut/data/annotation/JsonSubView.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2017-2023 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.data.annotation;
+import io.micronaut.context.annotation.AliasFor;
+import io.micronaut.core.annotation.Experimental;
+import io.micronaut.serde.annotation.Serdeable;
+import io.micronaut.data.annotation.JsonView.Operation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * The annotation defining Json Duality SubView. Currently supported only by Oracle database.
```

Comment:
need to elaborate this javadoc more. Describe what a subview is how it relates to the duality view feature in Oracle. Provide example code snippets etc.